tests/foreman/ui/test_reporttemplates.py

```diff
@@ -432,11 +435,14 @@ def test_positive_schedule_generation_and_get_mail(
         f'send "q\\r"\n'
     )
 
-    default_sat.execute(f"expect -c '{expect_script}'")
+    result = default_sat.execute(f"expect -c '{expect_script}'")
+    assert result.status == 0
     default_sat.get(remote_path=str(gzip_path), local_path=str(local_gzip_file))
-    os.system(f'gunzip {local_gzip_file}')
-    data = json.load(local_gzip_file.read_text())
-    subscription_search = default_sat.api.Subscription(organization=module_org).search()
+    result = os.system(f'gunzip {local_gzip_file}')
+    assert result == 0
+    text = local_file.read_text()
+    data = json.loads(text)
+    subscription_search = default_sat.api.Subscription(organization=module_manifest_org).search()
```

Comment:
Same here, combine steps ans u dont need unnecessary vars declaration thats not used more than once.
```suggestion
    assert os.system(f'gunzip {local_gzip_file}') == 0
    data = json.loads(local_file.read_text())
    subscription_search = default_sat.api.Subscription(organization=module_manifest_org).search()
```